### ISSUE TITLE
no-unknown-property: Unknown property 'download' found

### ISSUE THREAD

**Issue by hoonoh** [jsx-eslint/eslint-plugin-react#3393]
Hello,

It seems there is a false alarm with `a` tag's `download` attribute. This didn't happen with `v7.31.1`.

```html
<a download href="http://foo.com/bar.zip" >download foo</a>
```

**Comment by ljharb** [comment]
I see your commit; looking forward to the PR :-)
